src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,20 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [],
+};
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    checkStatus: (state) => {
+      const displayStatus = 'Under Construction';
+      state.push(displayStatus);
+    },
+  },
+});
```

Comment:
- [ ] Great work so far! However, according to the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions_v2.md), the reducer inside the categories slice needs to check the status and update the state. It appears that this step has been missed in your code, so please ensure that you first add a parameter called `action` in your reducer and check the payload from the action before making any updates to the state, Here is an example of how to do this:
```jsx
      state.categories =
        action.payload === "Under construction"
          ? "Under construction"
          : state.categories

```